### DISCUSSION TITLE
How to interop with stride engine by "Game"?

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#1432] [Q&A] [answered]
I am writting a UWP app,and try to use stride to control   3d part.
The 3d content would change the content to show following user's input.

**Comment by tebjan** [comment, marked as answer]
The current versions in the launcher (4.0 and 4.1) both support UWP, however with .NET 6 we will drop UWP as it will be no longer supported by Microsoft. Did you try 4.0? If yes, what did not work?
